tests/TestCaseBase.php

```diff
@@ -8,19 +8,6 @@
 
 abstract class TestCaseBase extends TestCase
 {
-    /**
-     * Useful to test code with no observable behaviour other than not crashing.
-     *
-     * In contrast to PHPUnit's native method, this lets the test case count towards coverage.
-     *
-     * @see TestCase::expectNotToPerformAssertions()
-     */
-    public static function assertDidNotCrash(): void
```

Comment:
I like having this named method better than the many repeated calls to `self::addToAssertionCount(1);`. We can replace the implementation, but I would like to keep the semantics.